src/common/transformations/src/transformations/common_optimizations/ric_fusion.cpp

```diff
@@ -0,0 +1,607 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/common_optimizations/ric_fusion.hpp"
+#include "transformations/utils/utils.hpp"
+
+#include <memory>
+#include <utility>
+#include <vector>
+
+#include <ngraph/opsets/opset8.hpp>
```

Comment:
can it use ov namespace instead of 'ngraph'?

Reply:
We agreed with @ilyachur that as we still have ngraph for all transformation we can use it until global transition.